firefox-ios/Client/Frontend/Browser/Tabs/Middleware/TabManagerMiddleware.swift

```diff
@@ -500,34 +475,6 @@ class TabManagerMiddleware: BookmarksRefactorFeatureFlagProvider,
         store.dispatch(action)
     }
 
-    /// Handles undoing the close tab action, gets the backup tab from `TabManager`
-    private func undoCloseTab(state: AppState, uuid: WindowUUID) {
-        toastTelemetry.undoClosedSingleTab()
```

Comment:
Reviewing the PR I just remember this telemetry was added recently, I'm checking with Andy if we should wait to remove this toast